### DISCUSSION TITLE
Could not connect to OctoPrint: curl:Couldn't connect to server:

### DISCUSSION THREAD

**Discussion by martin-badin** [OrcaSlicer/OrcaSlicer#7355] [Q&A]
Hello. I am using Orca Slicer few months. Everything worked before but currently I am not able to connect to the printer. I didn't change anything. I am using the latest version of OrcaSlicer (2.2.0). It works on the previous version. Connection thought web browser works.

Is there some bug or something was changed?

<img width="572" alt="Screenshot 2024-11-03 at 14 51 52" src="https://github.com/user-attachments/assets/3bfa3ad2-4044-4117-abe8-8b4f3b264e91">

**Comment by martin-badin** [comment]
Update: It works now but I had to install previous version, then test connection and then install the latest version.

**Comment by r3cn3p5** [comment]
Had this issue driving me mad, then on the Mac under settings/privacy security section, disabled and enabled OrcaSlicer , and started working

<img width="730" alt="image" src="https://github.com/user-attachments/assets/eabc365d-dea3-4488-be4f-e7de6f5a68d4">


Mac OSX: 15.0.1

**Reply by brdi** [reply]
Halleluja It worked for me as well. Stange hickup but Thanks for sharing

**Reply by bellisonoroff** [reply]
had the same issue and this worked for me as well. thanks for posting.

**Reply by byyasar** [reply]
Thanks,it s worked for me
